packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -33,227 +45,359 @@
   color: $bpk-color-white;
   box-shadow: none;
 
-  @include breakpoint-landscape {
-    &__sponsor-label {
-      margin-bottom: bpk-spacing-sm();
+  @include bpk-breakpoint-small-tablet {
+    @include aspect-ratio(133%);
+  }
 
-      @include bpk-label-1;
-    }
+  @include bpk-breakpoint-mobile {
+    @include aspect-ratio(133%);
+  }
 
-    &__sponsor-content {
-      margin-bottom: bpk-spacing-xl();
+  @include bpk-breakpoint-small-mobile {
+    @include aspect-ratio(133%);
+  }
+
+  &::after {
+    display: none;
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+
+    @include bpk-breakpoint-above-tablet {
+      height: 28.75rem;
+      justify-content: center;
+
+      @include bpk-margin-leading(bpk-spacing-xxxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xxl(), false);
+
+      &--start {
+        align-items: start;
+      }
+
+      &--center {
+        align-items: center;
+
+        /* adjusting for 1.5rem difference between margins */
+        > * {
+          @include bpk-margin-trailing(bpk-spacing-lg());
+        }
+      }
+
+      &--end {
+        align-items: end;
+      }
```

Comment:
Not 100% this suggestion would work but:

Would it be possible to split `start`, `middle` and `end` into their own rules, and combine them on the element such as `class="bpk-graphic-promo__container start"`

The idea here would that it may be possible to eliminate the duplication further down in this file? e.g. L100-116.

Reply:
Backpack uses BEM, so that would go against the rules, I am afraid.

Reply:
I really hate the duplication and the resetting overrides I had to make, but this is because of the way the new breakpoints are unfortunately. An alternative would be to use mixins, but then not everything can be a mixin, so I tried to put the line somewhere. 🙂

Reply:
>Backpack uses BEM, so that would go against the rules, I am afraid.

This is correct!

Let me check what we may have done for other components there might be something else we can do here!

Reply:
I'm happy with this if there isn't a reasonable solution to this 👍

Reply:
I did the same as the other suggestions below. 🙂

Reply:
I simplified the code a bit, based on the other suggestions. 🙂